### DISCUSSION TITLE
Lockdown feature not working

### DISCUSSION THREAD

**Discussion by jerlyn06** [kimai/kimai#3799] [Questions & Answers] [answered]
Hi,

I want to only allow time entries for today only and I set the start and end lockdown as follows. But I am still able to add begin dates preceding today. Can someone guide me what should be the lockdown start and end?
![image](https://user-images.githubusercontent.com/49563398/215030702-6ac8babf-296f-4f93-b41e-3929d61c5502.png)

**Comment by kevinpapst** [comment, marked as answer]
Read again how the configs work please: https://www.kimai.org/documentation/configurations.html#lockdown-period

Try start: `first day of last month 00:00:00` and end = `yesterday 23:59:59` and then activate the rule to prevent future time records.

**Comment by jerlyn06** [comment]
Hi Kevin, your example would allow entries in the last 1 month. However, my ask is different. I want to only allow time entries for today only. For example, if my current date is Jan 30, 2023, when adding timesheet entry I should **not** be able to choose dates other than Jan 30 in the 'From' field if that makes sense.

**Reply by kevinpapst** [reply]
Did you try it?

**Reply by jerlyn06** [reply]
Yes, I tried your answer. It allows me to add entries up to a month relative to 'now' - so it gives an error if I try to choose the 'From' date as anything before Nov 30, 2022. I am having trouble modifying the same limits to only today.

**Reply by kevinpapst** [reply]
<img width="921" alt="Bildschirm­foto 2023-01-30 um 14 39 04" src="https://user-images.githubusercontent.com/533162/215492785-8733cc5c-83ad-4745-92eb-1ac5c759032a.png">
<img width="561" alt="Bildschirm­foto 2023-01-30 um 14 39 49" src="https://user-images.githubusercontent.com/533162/215493079-c5c191a3-fde6-4cd1-8c3c-024ec05b2acd.png">
